generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -301,10 +312,18 @@ namespace nidaqmx_grpc {
       auto trigger_sources = request->trigger_sources().c_str();
       auto trigger_slope_array = reinterpret_cast<const int32*>(request->trigger_slope_array().data());
       auto trigger_level_array = const_cast<const float64*>(request->trigger_level_array().data());
-      if (request->trigger_slope_array().size() != request->trigger_level_array().size()) {
-        return ::grpc::Status(::grpc::INVALID_ARGUMENT, "The sizes of repeated fields trigger_slope_array and trigger_level_array do not match");
+      auto array_size_determine_from_sizes = std::array<int, 2>
```

Comment:
The "no optional" version of the code is not an improvement on the original. But I figure it's better to have one implementation of this check.